scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1146,8 +1278,7 @@ private void build() {
          * this will fail if someday the XML schema gets another <variable> tag
          * elsewhere
          */
-        Document dom = XMLParser.parse(jobDescriptor);
-        NodeList variables = dom.getElementsByTagName("variable");
+        NodeList variables = XMLParser.parse(jobDescriptor).getElementsByTagName("variable");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "variables" which hides the field declared at line 237. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)